src/behaviours/extension-special-characters-in-page-registrations.test.tsx

```diff
@@ -23,7 +23,7 @@ describe("extension special characters in page registrations", () => {
       extensionWithPagesHavingSpecialCharacters,
     );
 
-    await applicationBuilder.addExtensions(testExtension);
+    applicationBuilder.extensions.renderer.enable(testExtension);
```

Comment:
Have we already merged the other PR that fixes these calls to be `await`-ed?

Reply:
This PR makes `enable` synchronous.